controller/github.js

```diff
@@ -265,7 +265,7 @@ function grabLocalLogin() {
   if (!git) {
     throw new Error(`There is no file at ${userFilePath}.`);
   }
-  return git.login;
+  return git.username;
```

Comment:
Why was this change made? Does github use a different property name now? Also, I think this will break for existing users' credentials. You may want to try both for backward-compatibility:
```suggestion
  // Failover to `login` for backward-compatibility
  return git.username || git.login;
```

Reply:
When it was login it was comming back as undefined.